include/common.h

```diff
@@ -69,6 +69,8 @@ struct nvnc_client {
 	struct nvnc_common common;
 	int ref;
 	struct stream* net_stream;
+	char hostname[128];
+	char username[64];
```

Comment:
I used 256 for the maximum username length elsewhere. It's a bit ridiculous, but let's just keep it consistent.